### PR TITLE
refactor: move simple result to presentation

### PR DESCRIPTION
**Pull request by jsolaas**
Also reverse the imports/dependency

### CODE REVIEW

File: src/libecalc/presentation/simple_result/simple.py [hunk 1]
```diff
@@ -133,6 +134,10 @@ class SimpleComponentResult(SimpleBase):
     energy_usage_unit: Unit
     power: Optional[List[opt_float]]
 
+    @classmethod
+    def from_dto(cls, component_result) -> "SimpleComponentResult":
```
Comment (TeeeJay) [inline src/libecalc/presentation/simple_result/simple.py:138]: typehint

File: src/tests/libecalc/dto/results/test_simple.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from libecalc.common.component_info.component_level import ComponentLevel
 from libecalc.common.units import Unit
 from libecalc.dto.base import ComponentType
-from libecalc.dto.result.simple import (
+from libecalc.presentation.simple_result.simple import (
```
Comment (TeeeJay) [inline src/tests/libecalc/dto/results/test_simple.py:8]: this test should be moved to tests/libecalc/presentation? I know I havent been consistent and updated tests in general based on the rules we want to adhere to, just to check if you thought of that

File: src/libecalc/presentation/simple_result/simple.py (hunk 1 above)
Comment (jsolaas) [reply src/libecalc/presentation/simple_result/simple.py:138]: So a protocol then? BaseModel from pydantic? Import type from dto.results?

Comment (jsolaas) [reply src/libecalc/presentation/simple_result/simple.py]: Nevermind, I was thinking simple shouldn't import from dto, but that is what I'm fixing here..

**Pull request merged**